src/main/java/com/fasterxml/jackson/core/sym/ByteQuadsCanonicalizer.java

```diff
@@ -1096,13 +1096,17 @@ private int _resizeAndFindOffsetForAdd(int hash) throws StreamConstraintsExcepti
         return offset;
     }
 
+    static int multiplyByFourFifths(int number) {
```

Comment:
Let's not do this -- this is not performance critical in any way, no point optimizing in this case (whereas `NumberOutput` is indeed performance sensitive)

Reply:
The technique is used [in this project](https://github.com/FasterXML/jackson-core/blob/dc5b746f33aa96235040c291d910da15692504cd/src/main/java/com/fasterxml/jackson/core/io/schubfach/FloatToDecimal.java#L548) also among many [others](https://github.com/plokhotnyuk/jsoniter-scala/blob/7b3de2176d0e15457933e87d0080914545bca08a/jsoniter-scala-core/jvm/src/main/scala/com/github/plokhotnyuk/jsoniter_scala/core/JsonReader.scala#L2995). I have also verified this fact via my own benchmarks.